work/backports.rst

```diff
@@ -2,81 +2,73 @@
 Backporting Tickets to a Release Branch
 #######################################
 
-So you have a ticket that you think should be backported to ``v23.0.0`` and/or be used for DP0.2.
-What next?
+So you have a ticket that you think should be backported to the release branch? What next?
+Note, these instructions use v88 as an alias for the release of the science pipelines to backport to.
+Replace the 88 with 24, 27, or whatever major release number we are on now.
 
-1) First, add the label "backport-v23" to your jira ticket.
+1) First, add the label "backport-v88" to your jira ticket.
 
-   This will flag it for review by the DM-CCB or the DP0.2 campaign managers.
+   This will flag it for review by the DM-CCB.
    Continue merging your ticket to the default branch (``main``) and mark the ticket ``Done`` per instructions in the normal :doc:`flow`.
 
-2) Wait for approval. The ticket will gain the label "backport-approved."  A comment will be posted on the ticket that you may start the backporting process.
+2) Wait for approval. The ticket will gain the label "backport-approved."
 
-3) Checkout the release branch, ``v23.0.x``, for each repo affected by your ticket.
+3) Checkout the release branch, ``v88.0.x``, for each repo affected by your ticket.
 
    .. code-block:: bash
 
-      git checkout v23.0.x
+      git checkout v88.0.x
 
-   If the repo does not already have a ``v23.0.x`` branch, you need to create one based on ``v23.0.0.rc1`` (not the latest rcN, although that should be identical):
+   If the repo does not already have a ``v88.0.x`` branch, you need to create one based on ``v88.0.0.rc1`` (not the latest rcN, although that should be identical):
 
    .. code-block:: bash
 
-      git checkout v23.0.0.rc1
-      git checkout -b v23.0.x
-      git push -u origin v23.0.x
+      git checkout v88.0.0.rc1
+      git checkout -b v88.0.x
+      git push -u origin v88.0.x
 
-   Now treat ``v23.0.x`` same as you would the default branch (``main``).
+   Now treat ``v88.0.x`` same as you would the default branch (``main``).
 
-4) Create a copy of your ticket branch called ``tickets/DM-XXXXX-v23``.
+4) Create a copy of your ticket branch called ``tickets/DM-XXXXX-v88``.
 
    .. code-block:: bash
 
       git checkout tickets/DM-XXXXX
```

Comment:
This step would require to restore a deleted branch in case this ticket was already merged into `main`